webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-list.js

```diff
@@ -10,16 +10,41 @@ const rubricTitle = 'title';
 const rubricCreator = 'creator';
 const rubricModified = 'modified';
 
+const enableWeightsProperty = 'rubrics.enable.weighted';
+
 export class SakaiRubricsList extends RubricsElement {
 
   static get properties() {
 
     return {
       token: { type: String },
       rubrics: { type: Array },
+      weightedIsEnabled: { type: Boolean },
     };
   }
 
+  constructor() {
+    super();
+
+    this.weightedIsEnabled = true;
+
+    const self = this;
+    this.fetchServerProperty(enableWeightsProperty).then((propValue) => {
+      self.weightedIsEnabled = `${propValue}` === "true";
```

Comment:
Why are you wrapping this in a template string? Can't you just go propValue === "true"?